cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySqlClient.java

```diff
@@ -0,0 +1,190 @@
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.junit.Assert;
+
+import java.sql.*;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+public class CloudMySqlClient {
+
+    private static final String database = PluginPropertyUtils.pluginProp("DatabaseName");
+    private static final String connectionName = PluginPropertyUtils.pluginProp("ConnectionName");
+
+    public static void main(String[] args) throws SQLException, ClassNotFoundException {
+        getCloudMysqlConnection();
+        //createSourceTable("myTable");
+//        createSourceTable("newTable");
+//        String[] tablesToDrop = {"newTable"};
+//        dropTables(tablesToDrop);
+        //System.out.println("done");
+
+    }
+
+    public static Connection getCloudMysqlConnection() throws SQLException, ClassNotFoundException {
+        Class.forName("com.google.cloud.sql.mysql.SocketFactory");
+        String instanceConnectionName = "cdf-athena:us-central1:sql-automation-test-instance";
+        String databaseName = "TestDatabase";
+        String Username = "v";
+        String Password = "v@123";
+        String jdbcUrl = String.format("jdbc:mysql:///%s?cloudSqlInstance=%s&socketFactory=com.google.cloud.sql.mysql.SocketFactory&user=%s&password=%s", databaseName, instanceConnectionName, Username, Password);
+        Connection conn = DriverManager.getConnection(jdbcUrl);
```

Comment:
Don't hardcode this string here. Add it in the plugin parameter properties file and pick it from there.